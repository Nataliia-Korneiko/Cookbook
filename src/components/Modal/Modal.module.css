.backdrop {
  display: flex;
  position: fixed;
  justify-content: center;
  top: 0;
  left: 0;
  width: 100vw;
  height: 100vh;
  background-color: var(--bg-color-backdrop);
  align-items: center;
  z-index: 10;
}

.modal {
  border: none;
  border-radius: 8px;
  background-color: var(--bg-color-white);
  background-image: url(../../assets/img/backdrops/backdrop-04.png);
  background-repeat: no-repeat;
  background-position: center;
}
