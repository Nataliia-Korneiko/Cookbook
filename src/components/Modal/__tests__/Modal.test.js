import React from 'react';
import { shallow } from 'enzyme';
import Modal from '../Modal';

describe('Modal component', () => {
  const props = {
    children: '',
    onClose: () => null,
  };

  const newModal = shallow(<Modal />);

  it('renders a properly', () => {
    expect(newModal).toMatchSnapshot();
  });

  it('renders two div', () => {
    expect(newModal.find('div')).toHaveLength(2);
  });
});
