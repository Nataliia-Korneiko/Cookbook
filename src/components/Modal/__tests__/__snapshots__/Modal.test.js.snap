// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal component renders a properly 1`] = `ShallowWrapper {}`;
